pkg/okta/okta.go

```diff
@@ -109,12 +120,18 @@ func newJwtVerifier(clientID string, issuer string) *jwtverifier.JwtVerifier {
 type oktaMiddlewareFactory struct {
 	handlers.HandlerBase
 	verifier *jwtverifier.JwtVerifier
+	codeGRT  string
 }
 
 // NewOktaAuthorizeMiddleware returns a wrapper for HandlerFunc to authorize with Okta
-func NewOktaAuthorizeMiddleware(base handlers.HandlerBase, clientID string, issuer string) func(http.Handler) http.Handler {
+func NewOktaAuthorizeMiddleware(base handlers.HandlerBase, clientID string, issuer string, ovrdGRT bool) func(http.Handler) http.Handler {
 	verifier := newJwtVerifier(clientID, issuer)
-	middlewareFactory := oktaMiddlewareFactory{HandlerBase: base, verifier: verifier}
+
+	jobCodeGRT := grtStandard
```

Comment:
I think this could be more clear actually if the param "ovrdGRT" was called "isProd" or something. It doesn't feel to me like we're overriding anything, rather we're just choosing which job code is relevant from the environment

Reply:
If we pursue this avenue, the job code variables could be renamed to ProdGRTJobCode.

Reply:
Will do